recipes/libnvshmem/build-libnvshmem.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+$BUILD_PREFIX/bin/check-glibc bin/* lib/* bin/examples/* bin/perftest/device/coll/* bin/perftest/device/pt-to-pt/* bin/perftest/host/coll/* bin/perftest/host/init/* bin/perftest/host/pt-to-pt/*
```

Comment:
"$BUILD_PREFIX/bin/check-glibc" If you need to full path for this tool to work it's because you're using this script in an unintended way. I suggest moving this call to check-glibc into `build.sh` and moving the dependency on `cf-nvidia-tools` to the top-level requirements.

The install steps can stay in this script.